### PR TITLE
Add preliminary ITS endpoint

### PR DESCRIPTION
**Pull request by elsiel23**
We need to provide an auth token. I've contacted Carl @ SIG to about this.

### CODE REVIEW

File: src/planscape/existing_projects/views.py [hunk 1]
```diff
@@ -1,16 +1,38 @@
-import requests, urllib
+import requests
+import urllib
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from django.views.generic.base import TemplateView
-import requests, urllib
+import requests
```
Comment (riecke) [inline src/planscape/existing_projects/views.py:6]: Nit: remove this line and the next, since lines 1 and 2 are duplicates.

**Review by riecke — COMMENTED** [review]
Can you add some tests in tests.py?  Doesn't have to be for the original API, but this will encourage writing of tests going forward.

File: src/planscape/existing_projects/views.py (hunk 1 above)
Comment (elsiel23) [reply src/planscape/existing_projects/views.py]: Done

**Comment by elsiel23** [comment]
How's this for now? We're returning an error, so not much to validate atm

File: src/planscape/existing_projects/tests.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
 from django.test import TestCase
+from django.urls import reverse
+import json
 
-# Create your tests here.
+class ITSTest(TestCase):
+        
+    def test_x(self):
```
Comment (riecke) [inline src/planscape/existing_projects/tests.py:7]: Nit: change the name to "test_token_missing".

**Review by riecke — APPROVED** [review]
Thanks for starting the tests.

**Review by riecke — APPROVED** [review]

**Pull request merged**